bin/generate_ci_matrix.py

```
@@ -39,10 +46,20 @@
   if not env_platform:
     print(f"Error: Could not determine platform for environment '{pio_env}'")
     exit(1)
+  board_level = cfg.get(f"env:{pio_env}", "board_level", default=None)
```

Flake8 (7.3.0)

[error] indentation is not a multiple of 4

(E111)